automation/services/watchdog/watchdog.py

```diff
@@ -31,6 +31,8 @@ def main():
   prover_errors = Counter('Coda_watchdog_prover_errors', 'Description of gauge')
   pods_with_no_new_logs = Gauge('Coda_watchdog_pods_with_no_new_logs', 'Number of nodes whose latest log is older than 10 minutes')
   nodes_queried=Gauge('Coda_watchdog_nodes_queried', 'Number of nodes that were queried for node-status')
+  seed_nodes_responded=Gauge('Coda_watchdog_seed1_nodes_responded', 'Number of nodes that responded to the last status query on seed 1', ['seed'])
```

Comment:
change `Coda_watchdog_seed1_nodes_responded` to something like `Coda_watchdog_nodes_responded_to_seed`?